src/main/java/dk/kb/image/api/v1/impl/AccessApiServiceImpl.java

```diff
@@ -462,13 +462,13 @@ public static String getMIME(String format) {
      * @return ThumbnailsDto. Has a default thumbnail, a sprite and list of time sliced thumbnails.
      */
     @Override
-    public ThumbnailsDto kalturaThumbnails(String fileId, Integer numberOfThumbnails, Integer width, Integer height) throws  ServiceException {
+    public ThumbnailsDto kalturaThumbnails(String fileId, Integer numberOfThumbnails, Integer secondStartSeek, Integer secondEndSeek, Integer width, Integer height) throws  ServiceException {
      
         if (fileId == null) {
             throw new InvalidArgumentServiceException("FileId must not be null");
         }        
         try {
-           ThumbnailsDto thumbnails = KalturaUtil.getThumbnails(fileId, numberOfThumbnails, width, height);
+           ThumbnailsDto thumbnails = KalturaUtil.getThumbnails(fileId, numberOfThumbnails, secondStartSeek, secondEndSeek,width, height);
             return thumbnails;
         }
         catch(Exception e) {
```

Comment:
Would it make sense to make an overloading method with the old signature `public ThumbnailsDto kalturaThumbnails(String fileId, Integer numberOfThumbnails, Integer width, Integer height)` so that we are 100% backwards compatible?

Reply:
rejected